src/Files.Uwp/Views/Pages/PropertiesSecurityAdvanced.xaml.cs

```diff
@@ -93,26 +97,29 @@ private void Properties_Unloaded(object sender, RoutedEventArgs e)
         private async void AppSettings_ThemeModeChanged(object sender, EventArgs e)
         {
             var selectedTheme = ThemeHelper.RootTheme;
-            await Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
+            await DispatcherQueue.EnqueueAsync(() =>
             {
                 RequestedTheme = selectedTheme;
                 if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
                 {
                     switch (RequestedTheme)
                     {
                         case ElementTheme.Default:
-                            TitleBar.ButtonHoverBackgroundColor = (Color)Application.Current.Resources["SystemBaseLowColor"];
-                            TitleBar.ButtonForegroundColor = (Color)Application.Current.Resources["SystemBaseHighColor"];
+                            //WINUI3
+                            //TitleBar.ButtonHoverBackgroundColor = (Color)Application.Current.Resources["SystemBaseLowColor"];
```

Comment:
No longer supported?

Reply:
Supported with AppWindow.TitleBar - however it only works on w11

Reply:
Actually it is: `appWindow.TitleBar.ButtonHoverBackgroundColor` appears to work. Can be uncommented.

Reply:
We're using experimental so I think it should work

Reply:
I'll check but I think it still crashed on w10

Reply:
Fair enough, they're probably still working on backporting it but I think it's planned for the next stable release.